content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -111,81 +112,16 @@ following commands on a machine with cluster admin privileges:
     ISTIO_SERVICE_CIDR=172.21.0.0/16
     {{< /text >}}
 
-### Setup DNS
-
-Providing DNS resolution to allow services running on VM can access the
-services running in the cluster. Istio itself does not use the DNS for
-routing requests between services. Services local to a cluster share a
-common DNS suffix(e.g., `svc.cluster.local`). Kubernetes DNS provides
-DNS resolution for these services.
-
-To provide a similar setup to allow services accessible from VMs, you name
-services in the clusters in the format
-`<name>.<namespace>.global`. Istio also ships with a CoreDNS server that
-will provide DNS resolution for these services. In order to utilize this
-DNS, Kubernetes' DNS must be configured to `stub a domain` for `.global`.
-
-{{< warning >}}
```

Comment:
this warning should be kept in place or modified to indicate `istioctl` may not setup DNS properly in all cases.  `istioctl` should be verified that it handles all cloud providers properly.  There are the kube-dns and coredns plugin cases, and some cloud providers such as Microsoft require an additional configmap to stub.